appveyor.yml

```diff
@@ -6,6 +6,8 @@ skip_non_tags: true
 artifacts:
   - path: godot\bin\*.exe
     name: GodotBinary
+  - path: godot\bin\*.dll
```

Comment:
I think this will glob more than just the GDNative wrapper, it might be better to use `*gdnative_wrapper_code*.dll`